server/services/omuCaselist.test.ts

```diff
@@ -0,0 +1,187 @@
+import { add, sub, format } from 'date-fns'
+import OmuCaselist from './omuCaselist'
+import LicenceStatus from '../enumeration/licenceStatus'
+import LicenceType from '../enumeration/licenceType'
+import Container from './container'
+
+describe('omu caselist', () => {
+  it('should return prison view case', () => {
+    const today = format(new Date(), 'yyyy-MM-dd')
+    const omuCaselist = createCaselist(LicenceStatus.SUBMITTED, today, '2022-10-19')
```

Comment:
meh.. as you don't vary today you could extract it to the top level of your module and then you wouldn't pass it through everywhere - reinforcing that sharp focus to the relevant parts that you are testing

Reply:
done